### PR TITLE
Adding support for Azure SQL DB Transparent Data Encryption with Azur…

### PR DESCRIPTION
**Pull request by mihymel**
## Description
The changes include the newly generated hydra clients with support for the Azure SQL DB feature Transparent Data Encryption with Azure Key Vault integration. Tests are included.

The pull request to the corresponding hydra spec changes is here: https://github.com/Azure/hydra-specs-pr/pull/1366 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/dev/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@mihymel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mihymel). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by akromm** [comment]
LGTM.

**Comment by markcowl** [comment]
@akromm Can you review?

**Review by akromm-zz — APPROVED** [review]
Reviewed internally.

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@akromm Wow, that was fast!

**Comment by markcowl** [comment]
@mihymel Please include a link to the PR in hydra-specs-pr for this change

**Comment by mihymel** [comment]
I modified the initial description to include a link the the hydra-specs-pr PR. Let me know if I should include the link somewhere else as well.

File: src/ResourceManagement/Sql/NuGet.Config [hunk 1]
```diff
@@ -1,6 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-    <config>
-        <add key="repositorypath" value="..\..\..\packages" />
-    </config>
-</configuration>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (markcowl) [inline src/ResourceManagement/Sql/NuGet.Config]: should not be necessary.

**Comment by markcowl** [comment]
@mihymel I don't see the hydra-specs-pr link.  There are several modeling issues in this PR, so hydra-specs-pr changes will likely be necessary.

**Comment by mihymel** [comment]
@markcowl The hydra-specs-pr PR is here https://github.com/Azure/hydra-specs-pr/pull/1366

File: src/ResourceManagement/Sql/NuGet.Config (hunk 1 above)
Comment (mihymel) [inline src/ResourceManagement/Sql/NuGet.Config]: Visual Studio must have updated this file and I included the change on accident. I reverted the line endings back to UNIX with commit 0f11fccbbe3b8d14d330aaa299c2820cbf7dc771.

**Comment by mihymel** [comment]
@markcowl  Have you had the chance to look at the hydra-specs-pr changes here https://github.com/Azure/hydra-specs-pr/pull/1366 ? Let me know if you have any feedback.

**Comment by cormacpayne** [comment]
@mihymel can you please pull in the latest changes from master?

**Comment by cormacpayne** [comment]
LGTM once last changes from master have been pulled in

**Comment by mihymel** [comment]
@cormacpayne I rebased my fork to the latest from master.

**Pull request merged**